pos_order_return/README.rst

```diff
@@ -0,0 +1,102 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+================
+PoS Order Return
+================
+
+This module extends the functionality of odoo Point Of Sale about POS Order
+returns.
+
+With this module, it is now forbidden to return more quantity than the initial
+one.
+
+A link is created between the returned Order and the initial Order.
+A link is created between the returned Order Line and the initial Order Line.
+
+Usage
+=====
+
+Select an PoS Order an choose either *Return Products* (full return of the
```

Comment:
s/Select/Select in the backend

I expected to make a return from the POS and not need to go to the backend to return and order! Is it possible?

cc @legalsylvain

Reply:
It's possible. I'm working on a superseed of pos_order_list (yet to publish) to include it as a dependency here

Reply:
> I expected to make a return from the POS and not need to go to the backend to return and order! Is it possible?

> It's possible. I'm working on a superseed of pos_order_list (yet to publish) to include it as a dependency here

What is pos_order_list ? I don't see it in OCA/pos.

In the meantime, I worked (V8) on a module to list the done pos orders. For the time being, it display on PoS front UI the list of the past done / invoiced orders and allow to reprint the bill, but allowing to make a refund from this module can be easely done.

See the current module here (GRAP incubator repository), but I can make a PR against OCA/pos, if OCA members are interested by this module.

https://github.com/grap/grap-odoo-incubator/tree/8.0/pos_done_order_load

(see roadmap parts).

regards.

Reply:
I meant this PR: https://github.com/OCA/pos/pull/152 I think it can be a good base to extend from (has a little bug yet though).